executor/executor_test.go

```diff
@@ -1667,13 +1667,12 @@ func (s *testSuite) TestMiscellaneousBuiltin(c *C) {
 		}
 	}
 }
-
 func (s *testSuite) TestSchemaCheckerSQL(c *C) {
 	defer testleak.AfterTest(c)()
 	store, err := tikv.NewMockTikvStore()
 	c.Assert(err, IsNil)
 	tidb.SetStatsLease(0)
-	lease := 5 * time.Millisecond
+	lease := 20 * time.Millisecond
```

Comment:
Make sure the SQL of `alter table t modify column c bigint` is executed in a lease.